### ISSUE TITLE
Floating windows position isn't updated when the monitor layout changes

### ISSUE THREAD

**Issue by fdev31** [hyprwm/Hyprland#4765]
### Hyprland Version

0.35.0

### Bug or Regression?

Bug

### Description

When the monitor layout using `wlr-randr`:

Everything is updated to fit the new layout so there are no visible changes, but floating windows are not.

There is a chance to completely lose floating windows this way (when changing the monitor layout).

### How to reproduce

- Get a monitor at 0.0, eg: `wlr-randr --output HDMI-A-1 --pos 0,0`
- make some window floating
- Change to another position, eg: `wlr-randr --output HDMI-A-1 --pos 400,0` 
- see workspaces being updated, tiled windows are preserved, but floating windows are shifted horizontally (in the example)

**Comment by fdev31** [comment]
I don't think this is enough, also it's hard to operate on things you don't see, meaning you have to use such hacks/workarounds: https://github.com/hyprland-community/pyprland/wiki/lost_windows

or identify the client using hyprctl and then move it to the workspace of your choice using the command line... not very friendly

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
